kie-pmml-trusty/kie-pmml-models/kie-pmml-models-clustering/kie-pmml-models-clustering-tests/src/test/java/org/kie/pmml/clustering/tests/SingleIrisKMeansClusteringTest.java

```diff
@@ -28,8 +28,10 @@
 
     private static final String FILE_NAME = "SingleIrisKMeansClustering.pmml";
 
-    public SingleIrisKMeansClusteringTest(double sepalLength, double sepalWidth, double petalLength, double petalWidth, String irisClass, double outNormcontinuousField) {
-        super(sepalLength, sepalWidth, petalLength, petalWidth, irisClass, outNormcontinuousField);
+    public SingleIrisKMeansClusteringTest(
```

Comment:
As a by-side note: this overall design (SingleIrisKMeansClusteringTest and SingleIrisKMeansClusteringWithIdTest extending the same abstract class) makes them so bind that, actually, exactly the same test have to be duplicated on both classes, without any actual benefit. I think it would be better, in ano another PR, to redesign them slightly to allow adding test-case only on one of them without the need of modify the other. WDYT ?